.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
**Issue**: This project uses yarn instead of npm.

```suggestion
  - package-ecosystem: "yarn"
```

Reply:
Take a look over here : https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem 

package-ecosystem for yarn is also npm

Reply:
Ah, yeah I remember I ran into this before. Certainly very confusing.